TWLight/applications/forms.py

```diff
@@ -209,27 +209,25 @@ def _add_user_data_subform(self, user_data):
                 # Show which partner wants which personal data if applying
                 # for more than one.
                 if len(self.field_params) > 1:
+                    # fmt: off
                     # Translators: This text is shown in the application form under each piece of personal information requested. {partner_list} will be a list of 2 or more organisations that require this personal data, and should not be translated.
-                    self.fields[datum].help_text = _(
-                        "Requested by: {partner_list}".format(
+                    self.fields[datum].help_text = _("Requested by: {partner_list}".format(
                             partner_list=", ".join(user_data[datum])
                         )
```

Comment:
closing brackets for `_()` should be lifted to the same line as the source string

Reply:
Is this what you mean – the entire thing in one line? ```_("Requested by: {partner_list}".format(partner_list=", ".join(user_data[datum]))```

Reply:
You can also break the line on a `.`
```
_("Requested by: {partner_list}")
.format(partner_list=", ".join(user_data[datum]))
```